### PR TITLE
Create [WIP]2016-07-18-july-2016-economical-software-testing

### PR DESCRIPTION
**Pull request by hamoore**
@ryan-roemer or @scottnonnenberg, can one of you take a look and comment or approve? I'll get it posted as soon as it's ready! :)

### CODE REVIEW

**Comment by scottnonnenberg** [comment]
The copy looks good, and the syntax looks like it would work. But I haven't run it myself. :0)

File: [WIP]2016-07-18-july-2016-economical-software-testing.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+economical test suite.
+
+Dan Rathbone is an independent software engineer living and working in Seattle.  For the 
+past few years he's been building node.js web apps for companies large and small.  He's
+always happy to make new connections, so feel free to chat with him after the talk.
+
+
+## Venue / Sponsor
+
+**[Mixpo](http://mixpo.com/) has graciously agreed to host the event and provide
```
Comment (ryan-roemer) [inline [WIP]2016-07-18-july-2016-economical-software-testing.md]: Need to close bold, so:

```
**[Mixpo](http://mixpo.com/)**
```

File: [WIP]2016-07-18-july-2016-economical-software-testing.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
```
Comment (ryan-roemer) [inline [WIP]2016-07-18-july-2016-economical-software-testing.md]: NOTE: File needs to end in `.md` to render as markdown and be viewable here. I made the change already directly to this PR.

File: [WIP]2016-07-18-july-2016-economical-software-testing.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+layout: post
+title: "July 2016 Meetup - Economical Software Testing"
+description: ""
+tags: [node.js, javascript, software testing]
+---
+{% include JB/setup %}
+
+The Seattle Node.js Meetup is pleased announce our
+[meetup](xxx)
```
Comment (ryan-roemer) [inline [WIP]2016-07-18-july-2016-economical-software-testing.md]: TODO: Remember to update once event is live on Meetup.com!

File: [WIP]2016-07-18-july-2016-economical-software-testing.md [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+us via email at [{{site.author.email}}](mailto:{{site.author.email}}) or Twitter
+at [@{{site.author.twitter}}](https://twitter.com/{{ site.author.twitter }}).
+
+See you soon!
+
+## Agenda
+
+* **6:30 - 7:00pm**: Start with snacks and mingling, courtesy of Mixpo.
+
+* **7:00pm - 8:00pm**: Listen and learn about Economical Software Testing, as well
```
Comment (ryan-roemer) [inline [WIP]2016-07-18-july-2016-economical-software-testing.md]: Replace `Listen and learn about Economical Software Testing` with `Listen and learn about cost / expense analysis in software testing`

File: [WIP]2016-07-18-july-2016-economical-software-testing.md [hunk 5]
```diff
@@ -0,0 +1,53 @@
+---
+layout: post
+title: "July 2016 Meetup - Economical Software Testing"
+description: ""
+tags: [node.js, javascript, software testing]
```
Comment (ryan-roemer) [inline [WIP]2016-07-18-july-2016-economical-software-testing.md]: Change `software testing` to `testing`.

As a general note try to find an existing tag to match. You can find our existing tag collection at: http://seanode.github.io/tags

But this is totally ad hoc, so not a big deal really...

**Comment by ryan-roemer** [comment]
Great work!

A couple of small changes, after which rest LGTM! And you don't need my re-review to go ahead and publish to meetup, then finish and merge this one!

**Pull request merged**